src/Shared/Recruit.Shared.Web/Mappers/VacancySummaryMapper.cs

```diff
@@ -18,7 +18,7 @@ public static VacancySummaryViewModel ConvertToVacancySummaryViewModel(VacancySu
                 NoOfNewApplications = vacancySummary.NoOfNewApplications,
                 NoOfSuccessfulApplications = vacancySummary.NoOfSuccessfulApplications,
                 NoOfUnsuccessfulApplications = vacancySummary.NoOfUnsuccessfulApplications,
-                ClosingDate = vacancySummary.ClosingDate,
+                ClosingDate = vacancySummary.ClosedDate ?? vacancySummary.ClosingDate,
```

Comment:
The UI of the dashboard still shows `ClosingDate` which might make this confusing.